ECMA2Yaml/ECMAHelper/Transform/ECMADocsTransform.xsl

```diff
@@ -110,9 +110,15 @@
   </xsl:template>
 
   <xsl:template match="see[@href and not(parent::member)]">
-    <a>
-      <xsl:apply-templates select="@*|node()"/>
-    </a>
+    <xsl:choose>
```

Comment:
the task is actually about dealing with <see href="url"/> tags that are without any text. So we should check for `text()`, and use href as text instead if the original <see> tag does not have any text.

Reply:
could you also add test data for this?